google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/mutaterows/MutateRowsErrorConverterUnaryCallableTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub.mutaterows;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.api.core.SettableApiFuture;
+import com.google.api.gax.grpc.GrpcStatusCode;
+import com.google.api.gax.rpc.ApiExceptionFactory;
+import com.google.api.gax.rpc.UnaryCallable;
+import com.google.cloud.bigtable.data.v2.internal.RequestContext;
+import com.google.cloud.bigtable.data.v2.models.BulkMutation;
+import com.google.cloud.bigtable.data.v2.models.MutateRowsException;
+import com.google.cloud.bigtable.data.v2.models.MutateRowsException.FailedMutation;
+import com.google.cloud.bigtable.data.v2.stub.MutateRowsErrorConverterUnaryCallable;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+@RunWith(JUnit4.class)
+public class MutateRowsErrorConverterUnaryCallableTest {
+
+  private static final RequestContext REQUEST_CONTEXT =
+      RequestContext.create("fake-project", "fake-instance", "fake-profile");
+  private UnaryCallable<BulkMutation, MutateRowsAttemptResult> innerCallable;
```

Comment:
A better approach would be:

1. add mockito junit rule:
```@Rule public final MockitoRule mockitoRule = MockitoJUnit.rule()```
2. Annotate your mockito instance variables:
```
@Mock
private UnaryCallable<BulkMutation, MutateRowsAttemptResult> innerCallable;
@Captor 
private ArgumentCaptor<BulkMutation> innerMutation;
```

3. remove the Mockito.mock & ArgumentCaptor from setup

Reply:
Done